src/lavinmq/amqp/stream_consumer.cr

```diff
@@ -9,8 +9,8 @@ module LavinMQ
       property segment : UInt32
       property pos : UInt32
       getter requeued = Deque(SegmentPosition).new
-      getter filter : Array(String)? = nil
-      getter? match_unfiltered : Bool = false
+      @filter : Array(String)? = nil
```

Comment:
My last question is, should the filter be an empty array by default? Then  `filter_match?` could return early with `return true if @filter.empty?